packages/integrations/turbolinks/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@astrojs/turbolinks",
-  "description": "Turbolinks + Astro Integrations",
+  "description": "Astro + Turbolinks integration",
```

Comment:
```suggestion
  "description": "Deprecated — Use Turbolinks to speed up page navigation in your Astro site",
```

Reply:
Maybe we should even hide it from the Integrations catalogue? But sure, as a followup PR